src/bunyanWrapper.js

```diff
@@ -4,11 +4,12 @@
 var bunyan = require('bunyan');
 const cloneDeep = require('clone-deep');
 const constants = require('./utils/constants.js');
-var env = process.env.NODE_ENV;
+var env = process.env.SP_PRETTY_PRINT;
```

Comment:
@yogesh8177 change the name of variable from `env` to something else

Reply:
changed it to `shouldPrettyPrint`.